toolkit/imageconfigs/full-aarch64-64k.yaml

```diff
@@ -0,0 +1,124 @@
+storage: 
+  bootType: efi 
+  disks: 
+  - partitionTableType: gpt
+    maxSize: 8G
+    partitions:
+    - id: esp
+      type: esp
+      size: 9M
+    - id: boot
+      size: 200M
+    - id: root
+      size: grow
+  filesystems:
+  - deviceId: esp
+    type: fat32
+    mountPoint:
+      path: /boot/efi
+      options: umask=0077
+  - deviceId: boot
+    type: ext4
+    mountPoint:
+      path: /boot
+  - deviceId: root
+    type: ext4
+    mountPoint:
+      path: /
+os:
+  bootloader:
+    resetType: hard-reset
+
+  selinux:
+    mode: disabled
+
+  kernelCommandLine:
+    extraCommandLine:
+    - efi_mmap_nr_slack_slots=256
+    - console=tty0
+    - console=ttyS0,115200n8
+
+  packages:
+    install:
+    - kernel-64k
+    - kernel-64k-drivers-accessibility
+    - kernel-64k-drivers-sound
+    # iso required packages
+    - squashfs-tools
+    - tar
+    - device-mapper
+    - curl
+    - createrepo_c
+    - jq
+
+    installLists:
+    - packagelists/isoinstaller-64k-packages.yaml
+
+    remove:
+    - kernel
+
+  additionalFiles:
+  - source: ../out/tools/imager
+    destination: /installer/imager
+  - source: ../out/tools/liveinstaller
+    destination: /installer/liveinstaller
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/init
+    destination: /init
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/installer/calamares-EULA.txt
+    destination: /etc/calamares/azl-eula  
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/installer/terminal-EULA.txt
+    destination: /installer/EULA.txt
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/root/asoundrc
+    destination: /root/.asoundrc
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller
+    destination: /root/runliveinstaller
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/root/silence.wav
+    destination: /root/silence.wav
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/usr/lib/mariner/terminfo/mariner-installer
+    destination: /usr/lib/mariner/terminfo/m/mariner-installer
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/usr/lib/systemd/system/getty@.service
+    destination: /usr/lib/systemd/system/getty@.service
+  - source: ../resources/imageconfigs/additionalfiles/iso_initrd/usr/lib/systemd/system/serial-getty@.service
+    destination: /usr/lib/systemd/system/serial-getty@.service
+  - source: ../resources/manifests/image/local.repo
+    destination: /etc/yum.repos.d/mariner-iso.repo
+  - source: files/imagecustomizer/isoinstaller/attended_config_aarch64_64k.json
+    destination: /config/attended_config.json
+
+  users:
+    - name: root
+      passwordExpiresDays: 99999
+      startupCommand: /root/runliveinstaller
+
+iso:
```

Comment:
Suresh, we should probably move the `iso` section after the `scripts` section. That way, everything before `iso` is the rootfs image, and then the iso configuration. Just for the sake of organization.

Reply:
Updated.